modules/@apostrophecms/area/ui/apos/components/AposAreaContextualMenu.vue

```diff
@@ -176,6 +182,20 @@ export default {
       return flag;
     },
     myMenu() {
+      // Ensures we take a fresh look at the clipboard when toggled open again.
```

Comment:
All I really want to do is always touch "this.open" and "this.buttonOpen" in this `computed` function so that it responds to changes in their values. I did it this particular way to make sure neither developers nor compilers / optimizers ever decide it can be "optimized away."